src/PrestaShopBundle/Resources/config/services/core/grid/grid_data_factory.yml

```diff
@@ -95,10 +100,10 @@ services:
             - 'language'
 
     prestashop.core.grid.factory.language_decorator:
-        class: 'PrestaShop\PrestaShop\Core\Grid\Data\Factory\LanguageGridDataFactory'
-        arguments:
-            - '@prestashop.core.grid.data.factory.language'
-            - '@prestashop.adapter.language.language_flag_thumbnail_provider'
+      class: 'PrestaShop\PrestaShop\Core\Grid\Data\Factory\LanguageGridDataFactory'
```

Comment:
The indentation is four spaces in this file.

Reply:
Fixed